react/src/context/PostHogProvider.tsx

```diff
@@ -14,7 +14,7 @@ export function PostHogProvider({
     children?: React.ReactNode
     client?: PostHog | undefined
     apiKey?: string | undefined
-    options?: any | undefined
+    options?: Partial<PostHogConfig> | undefined
```

Comment:
Is options not exactly `PostHogConfig` ?

```suggestion
    options?: PostHogConfig | undefined
```

Reply:
Without Partial it would require them to enter all the keys of PostHogConfig. However, when setting the config it's fine to just set one of the options keys e.g. 
<img width="864" alt="Screenshot _app tsx — posthog-js (Code) 2023-03-14 at 10 53@2x" src="https://user-images.githubusercontent.com/22766134/224979571-caa1f927-b548-4b36-a923-ace21d9fcab3.png">